design-patterns/README.md

```diff
@@ -0,0 +1,33 @@
+# Design Patterns ![](https://badgen.net/badge/-/TypeScript/blue?icon=typescript&label)
```

Comment:
One thing I forgot: check in with @aanchalHS and Amelia on what the contents of this README should be.